libs/wingc/src/type_check.rs

```diff
@@ -1527,7 +1547,7 @@ impl<'a> TypeChecker<'a> {
 						return self.types.error();
 					}
 				} else {
-					self.spanned_error(callee, "Expected a function or method");
+					self.spanned_error(callee, format!("Expected a function or method got {}", func_type));
```

Comment:
I believe this is our usual format for this type of error:

```suggestion
					self.spanned_error(callee, format!("Expected a function or method, found \"{}\"", func_type));
```